Review JavaScript and static media folder

fpirola opened the pull request:
- replace http:// with :// ;
- review JavaScript function and naming convention;
- move and reorganize static media folder/file.

All changes has not been tested.

fpirola committed:
Review JavaScript and static media folder:
  - replace http:// with :// ;
  - review JavaScript function and naming convention;
  - move and reorganize static media folder/file.

All changes has not been tested.

fpirola committed:
Little Fix on base.html

kleptog commented:
I've finally had a chance to work through all the errors when building from scratch. It looks like it were getting stuck on the fact the Django 1.3 is pretty old. I upgraded Django to 1.6 and fixed everything that appeared to break, but now your patch conflicts. Could you try rebasing?

fpirola committed:
Resolved conflicts.

All changes has not been tested.

Merge branch 'master' of https://github.com/kleptog/DDTSS-Django

Conflicts:
	src/ddtp/ddtss/static/ddtss/ddtss.css
	src/ddtp/ddtss/static/ddtss/ddtss.js
	src/ddtp/media/ddtss/css/ddtss.css
	src/ddtp/media/ddtss/ddtss.css
	src/ddtp/media/ddtss/ddtss.js
	src/ddtp/media/ddtss/js/ddtss.js
	src/ddtp/templates/base.html
	src/ddtp/templates/ddtss/base_ddtss.html
	src/ddtp/templates/ddtss/index_lang.html
	src/ddtp/templates/milestones-lang.html
	src/ddtp/templates/one_milestones-lang.html

fpirola committed:
Review after some tests:
  - replace :// with // ;
  - fixed message.error(..., ...) - missing http request;
  - fixed "{% url" with non-empty first argument, see django documentation;
  - beatify: replace " with ' in "{% url", see django documentation, indentation.

fpirola committed:
Review after some tests:
  - replace :// with // ;
  - fixed call to message.error(..., ...) - missing http request;
  - fixed "{% url" with non-empty first argument, see django documentation;
  - beautify: replace " with ' in "{% url", see django documentation, indentation.

Merge branch 'master' of https://github.com/kleptog/DDTSS-Django

Conflicts:
	src/ddtp/ddtp_web/management/commands/generate_secret_key.py
